libs/mapjet-core/src/lib/utils/layer-event-handler.ts

```diff
@@ -61,6 +69,16 @@ export class LayerEventHandler {
     // Add the layers and features info to the event
     event.eventLayers = sortedLayers;
     event.eventFeatures = eventFeatures;
+    event.cancelBubble = false;
```

Comment:
to tylko tak aby o tym pamiętać, że jest deprecated ;)
https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble

Reply:
Good catch - zmienię ;)